device_detector/regexes/upstream/device/mobiles.yml

```diff
@@ -430,6 +430,18 @@ Apple:
     - regex: '(?:Apple-)?iPad7[C,][56]'
       model: 'iPad 6'
       device: 'tablet'
+    - regex: '(?:Apple-)?iPad8[C,][5678]'
+      model: 'iPad Pro 3 12.9'
+      device: 'tablet'
+    - regex: '(?:Apple-)?iPad8[C,][1234]'
+      model: 'iPad Pro 11'
+      device: 'tablet'
+    # - regex: '(?:Apple-)?iPad11[C,][34]'
+    #   model: 'iPad Air 3'
+    #   device: 'tablet'
+    # - regex: '(?:Apple-)?iPad11[C,][12]'
+    #   model: 'iPad Mini 5'
+    #   device: 'tablet'
```

Comment:
What are these commented out regexes for?

Reply:
These regexes were commented out at that time because the model number was not known yet. Will put another pull request including regexes for both devices